frontend/src/metabase/search/components/filters/CreatedByFilter/UserListElement/UserListElement.unit.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import userEvent from "@testing-library/user-event";
 import { renderWithProviders, screen } from "__support__/ui";
 import { createMockUserListResult } from "metabase-types/api/mocks";
-import { UserListElement } from "metabase/search/components/filters/CreatedByFilter/UserListElement";
+import { UserListElement } from "metabase/search/components/filters/CreatedByFilter/UserListElement/UserListElement";
```

Comment:
Ahhh. I think this might be the remnant from refactoring, you should be able to leave this as is (revert the change back)

Reply:
Smh. The auto-import from Webstorm strikes again